app/_hub/kong-inc/acme/index.md

```diff
@@ -38,7 +40,7 @@ params:
       required: false
       default: "`https://acme-v02.api.letsencrypt.org`"
       description: |
-        The ACMEv2 API endpoint to use. You can use the [Let's Encrypt staging environment](https://letsencrypt.org/docs/staging-environment/) during testing. Note that Kong doesn't automatically delete staging certificates: if you use same domain to test and use in production, you will need to delete those certificates manually after testing.
+        The ACMEv2 API endpoint to use. User might use [Let's Encrypt staging environemnt](https://letsencrypt.org/docs/staging-environment/)(`https://acme-staging-v02.api.letsencrypt.org/directory`) during testing. Note that Kong doesn't automatically delete staging certificates: if you use same domain to test and use in production, you will need to delete those certificates manually after testing.
```

Comment:
```suggestion
        The ACMEv2 API endpoint to use. The default is the [Let's Encrypt staging environment](https://letsencrypt.org/docs/staging-environment/) (`https://acme-staging-v02.api.letsencrypt.org/directory`), which may be used for testing. Note that Kong doesn't automatically delete staging certificates: if you use same domain to test and use in production, you will need to delete those certificates manually after testing.
```

Reply:
this might be incorrect, the default environment will be the default value for this field but not the staging one.

Reply:
Good point - I missed that.. Updated suggestion to reflect this.